### PR TITLE
Add Git for Windows and Nano recipes (both windows only recipes)

### PR DESCRIPTION
**Pull request by msarahan**
This is intended for usage at Software Carpentry workshops.  The idea is to be able to bootstrap a complete installation by only installing miniconda, then installing a conda environment: https://github.com/swcarpentry/swc-conda-recipes/pull/1

That PR has builds at https://anaconda.org/swc that were built using the recipes contained in this PR.

### CODE REVIEW

**Comment by ccordoba12** [comment]
You also have a recipe for `console_shortcut` here, but you just merged another PR with the same(?) recipe.

Please fix :-)

**Comment by ccordoba12** [comment]
By the way, great to have `nano` on Windows! We should also add recipes for clink and ConsoleZ too ;-)

**Comment by msarahan** [comment]
I'm not aware of ConsoleZ - I need to learn about it.  I have been intending to get Cmder (my favorite) into a recipe.  It rolls in clink.  Maybe better to have them separate, though?

**Comment by msarahan** [comment]
I will try to clean up console_shortcut, but cleaning up merge commits generally does not end well for me.

**Comment by ccordoba12** [comment]
I tried Cmder but it failed for me because i haven't the VS 2015 redistributable (I think). ConsoleZ is simpler, but it doesn't have that requirement.

**Comment by msarahan** [comment]
Git for windows has a small issue wherein git bash is not getting Library\bin for the environment on its PATH.  Needs a fix here.

File: console_shortcut/menu-windows.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "menu_name": "Python Prompts",
+    "menu_items":
+    [
+        {
+            "name": "Conda Env Prompt",
```
Comment (JanSchulz) [inline console_shortcut/menu-windows.json]: Shouldn't have this the env name?

Comment (msarahan) [inline console_shortcut/menu-windows.json]: Actually, that recipe shouldn't be in this PR at all.  Thanks for pointing it out.  I'll try to clean it up.

**Pull request closed without merging**